pkg/cfn/manager/api.go

```diff
@@ -110,13 +111,17 @@ func (c *StackCollection) DoCreateStackRequest(i *Stack, templateBody []byte, ta
 // any errors will be written to errs channel, when nil is written,
 // assume completion, do not expect more then one error value on the
 // channel, it's closed immediately after it is written to
-func (c *StackCollection) CreateStack(name string, stack builder.ResourceSet, tags, parameters map[string]string, errs chan error) error {
+func (c *StackCollection) CreateStack(name string, stack builder.ResourceSet, tags, parameters map[string]string, printer printers.OutputPrinter, errs chan error) error {
 	i := &Stack{StackName: &name}
 	templateBody, err := stack.RenderJSON()
 	if err != nil {
 		return errors.Wrapf(err, "rendering template for %q stack", *i.StackName)
 	}
 
+	if err := printer.LogObj(logger.Debug, "stack = \\\n%s\n", stack.Template()); err != nil {
```

Comment:
This logger is actually for Kubernetes-style objects, albeit it might work for other things too.

Reply:
thank you, didn't know, so what function should I use?

Reply:
if I understand correctly the code, `PrintObj` will just marshall and indent any object